### ISSUE TITLE
Ram eater

### ISSUE THREAD

**Issue by madwayz** [Marc3842h/Titan#96]
The reporter consumes a lot of RAM if you upload ~10 accounts or more. Make 2FA code could be phased. First one account to authorize, then the second. Not all at once the window opened.

**Comment by Marc3842h** [comment]
If I understood you correctly, you want it all "queue" like mode where one account bots after another finished its process. 

I've already thought about this a few months ago but decided against it because that would give out what Titan makes unique to other report bots: multi-threaded reporting. 

However considering that I'll implement Team Fortress 2 support in the future and that TF2 requires _alot_ of reports, this could very well be a thing that I'll implement in the future. 

I'll reevaluate this and ask the community for feedback on this suggestion.

**Comment by madwayz** [comment]
Yeah, you got me right. Unfortunately, I won't be able to use this software, because too often I change my Titan accounts and the computer will freeze. I will keep an eye on your GIT wait until everything is fixed. Good Luck :)
